src/main/java/forestry/storage/CrateRegistry.java

```diff
@@ -61,6 +61,9 @@ private static void registerCrate(ItemStack stack, @Nullable String oreDictName)
 
 	@Override
 	public void registerCrate(String oreDictName) {
+		if (ModuleCrates.cratesRejectedOreDict.contains(oreDictName)) {

```

Comment:
Can you please add a 'blacklistCrate' function to the 'ICrateRegistry' interface.